src/sentry/snuba/metrics/fields/snql.py

```diff
@@ -1016,9 +1016,13 @@ def on_demand_eps_snql_factory(
     return rate_snql_factory(aggregate_filter, interval, 1, alias)
 
 
+# Compare with discover:
+# https://github.com/getsentry/sentry/blob/4dc9c8b46538c1c4d8397344c9ccaf6ad8fd8d52/src/sentry/search/events/datasets/discover.py#L1506-L1553
 def on_demand_user_misery_snql_factory(
     aggregate_filter: Function, org_id: int, use_case_id: UseCaseID, alias: str | None = None
 ) -> Function:
+    # XXX: To my suprise, this is returning 6 miserable users rather than 4 users
+    # I may be setting things up incorrectly in the test
```

Comment:
I can determine that the value is 6 because I tinkered with the return value as you suggested.
```diff
-    return Function(
-        "divide",
-        [
-            Function("plus", [miserable_users, constants.MISERY_ALPHA]),
-            Function("plus", [unique_users, (constants.MISERY_ALPHA + constants.MISERY_BETA)]),
-        ],
-        alias=alias,
-    )
+    return Function("plus", [miserable_users, 0], alias=alias)
```